src/java.base/share/classes/sun/security/util/DerOutputStream.java

```diff
@@ -482,6 +482,17 @@ private DerOutputStream writeString(String s, byte stringTag, Charset charset) {
         return this;
     }
 
+    /**
+     * Takes a Date and chooses UTC or GeneralizedTime as per RFC 2630
+     */
+    public DerOutputStream putTime(Date d) {
+        @SuppressWarnings("deprecation")
+        Date low = new Date(1950,1,1);
```

Comment:
Isn't the form of the `Date` constructor deprecated?  Alternately you could convert 1/1/1950 into epoch time as a long and use `new Date(long)` which is not deprecated.  Same for line 492.